z-way.coffee

```diff
@@ -50,6 +50,10 @@ module.exports = (env) ->
         configDef: deviceConfigDef.ZWayLuminiscenceSensor,
         createCallback: (config) => new ZWayLuminiscenceSensor(config)
       })
+      @framework.deviceManager.registerDeviceClass("ZWayHumidSensor", {
+        configDef: deviceConfigDef.ZWayLuminiscenceSensor,
+        createCallback: (config) => new ZWayLuminiscenceSensor(config)
```

Comment:
Please create a new class for this new kind of sensor. Do not "misuse" a similar class!